source/js/storage.js

```diff
@@ -0,0 +1,179 @@
+const userInfo = {};
+const renderedText = {};
+
+window.addEventListener('DOMContentLoaded', init);
```

Comment:
Codacy has a fix for the issue: [Replace `'DOMContentLoaded'` with `"DOMContentLoaded"`](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8429666)
```suggestion
window.addEventListener("DOMContentLoaded", init);
```